zigzag/zigzag.py

```diff
@@ -85,17 +95,13 @@ def _generate_test_logs(junit_xml):
         test_log.module_names = [testsuite_props['RPC_PRODUCT_RELEASE']]         # RPC Release Codename (e.g. Queens)
         test_log.exe_start_date = date_time_now.strftime('%Y-%m-%dT%H:%M:%SZ')   # UTC timezone 'Zulu'
         test_log.exe_end_date = date_time_now.strftime('%Y-%m-%dT%H:%M:%SZ')     # UTC timezone 'Zulu'
+        test_log.automation_content = testcase_xml.find("./properties/property/[@name='test_id']").attrib['value']
         test_log.attachments = \
             [swagger_client.AttachmentResource(name="junit_{}.xml".format(date_time_now.strftime('%Y-%m-%dT%H-%M')),
                                                content_type='application/xml',
                                                data=b64encode(serialized_junit_xml).decode('UTF-8'),
                                                author={})]
 
-        try:
-            test_log.automation_content = testcase_xml.find("./properties/property/[@name='test_id']").attrib['value']
-        except AttributeError:
```

Comment:
Do we need to update the XSD to make the test_id property required? https://github.com/rcbops/pytest-rpc/blob/master/data/molecule_junit.xsd#L23

Reply:
So yeah...

After looking at the code again it turns out you could get malformed XML past zigzag. To top it off I forgot to enable capturing of the `start_time` and `end_time` that is now available in the JUnitXML. 🤦‍♂️